### PR TITLE
Change location of generated documentation

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Sets location for documentation generated
by Sphinx to top level directory `sphinx-build`

Simplifies the `clean-docs-all` target as only
the `sphinx-folder` folder needs to be removed

Replaces all docs/\* entries in .gitignore
with the `sphinx-build`

Addresses #36

### CODE REVIEW

File: docs/include.am [hunk 1]
```diff
@@ -7,32 +7,33 @@
 
 SPHINXOPTS    = ${SPHINX_WARNINGS} -q
 PAPER         =
-SPHINX_BUILDDIR      = ${abs_srcdir}/docs
+SPHINX_SOURCEDIR = ${abs_srcdir}/docs
+SPHINX_BUILDDIR      = ${abs_srcdir}/sphinx-build
```
Comment (ben-palmer-sociomantic) [inline docs/include.am]: Weird alignment of this line.

**Comment by ben-palmer-sociomantic** [comment]
Apart from my comment about alignment, LGTM.

**Comment by andreas-bok-sociomantic** [comment]
Fixup alignment and rebased on latest master 

``` diff
diff --git a/docs/include.am b/docs/include.am
index dfcfa6c..f8df682 100644
--- a/docs/include.am
+++ b/docs/include.am
@@ -8,7 +8,7 @@
 SPHINXOPTS    = ${SPHINX_WARNINGS} -q
 PAPER         =
 SPHINX_SOURCEDIR = ${abs_srcdir}/docs
-SPHINX_BUILDDIR      = ${abs_srcdir}/sphinx-build
+SPHINX_BUILDDIR  = ${abs_srcdir}/sphinx-build

```

**Comment by andreas-bok-sociomantic** [comment]
@ben-palmer-sociomantic, if this looks good to you, feel free to merge

**Comment by ben-palmer-sociomantic** [comment]
This pull request has been rebased via `git hub pull rebase`. Original pull request HEAD was bef46c3c424dd14bce25f52b6ae8a39befbf5952, new (rebased) HEAD is df8425e839b2b3f0690c9d8c5a422426b5fc142e

**Comment by andreas-bok-sociomantic** [comment]
Great TX!

**Pull request closed without merging**